### ISSUE TITLE
Missing stats from Community card in the Discover Communities screen

### ISSUE THREAD

**Issue by Francesca-G** [status-im/status-react#21233]
# Design Feedback 2.30

## Missing stats from Community card in the Discover Communities screen
In the Discover Communities screen cards are missing the members and active users numbers, resulting in just an icon with no purpose of being there.

Figma review [here](https://www.figma.com/design/oznddnBnDbLlLQIaACYNuj?node-id=2227-55758#938008754)

➡️ if this is not implemented yet we should remove the icons entirely, there's no purpose in having them there in the meantime
➡️ if it's a fetching issue we should use the stat loading state

#### Expected behavior

<img width="300" alt="Screenshot 2024-09-09 alle 15 44 07" src="https://github.com/user-attachments/assets/9ac8fd5d-4a54-4e0e-ba28-48d9493b10ac">

#### Actual behavior

<img width="300" alt="Screenshot 2024-09-09 alle 15 45 05" src="https://github.com/user-attachments/assets/f09d43e9-27f7-4bc4-9955-e5c3931b6815">

### Additional Information

- Operating System: Android, iOS
- Devices in the screenshots: iPhone 13

**Comment by churik** [comment]
I don't think that we have a capacity for it in 2.32, given the desired release cut date.
Deferring the issue, please be free to move it bak if you think there is a room for it @ilmotta

**Comment by churik** [comment]
it is shown in Desktop:

![Image](https://github.com/user-attachments/assets/3d6ebbcd-326b-4a3d-98ef-09c81454c2d1)

**Comment by churik** [comment]
We should either remve the icon or display the stat

**Comment by ulisesmac** [comment]
@churik @ilmotta 
There are some areas of improvement for communities, but not sure if communities are a priority right now

**Comment by ilmotta** [comment]
> [@churik](https://github.com/churik) [@ilmotta](https://github.com/ilmotta) There are some areas of improvement for communities, but not sure if communities are a priority right now

Communities for mobile are not a priority during this release cycle 2.33 and probably 2.34 @ulisesmac.

That doesn't mean we shouldn't do anything regarding communities. @churik and I talked about this issue today and that we should do something about it in 2.34 if possible. Broken features should be either fixed or removed/disabled. Given that the desktop app has this feature implemented, it likely means it's mostly a client-side implementation on mobile. But the simpler option is not out of question either, which is to simply disable the empty stats behind a flag.

If you notice areas of improvement in communities which aren't mapped in GH issues, please feel free to create issues and leave them in the state `Backlog` under the Mobile Core project, that's really helpful 💯
